DataTables.Blazor.Demo/Pages/Index.razor

```diff
@@ -306,6 +343,30 @@ window.demo = {
 </div>
 
 @code{
+    DataTableOptions _buttonsDemoOptions;
+
+    protected override void OnInitialized()
+    {
+        _buttonsDemoOptions = new DataTableOptions
+        {
+            DOM = "Bfrtip",
+            Buttons = new List<object>
+            {
+                "colvis"
+            },
+            Language = new LanguageOptions
+            {
+                Buttons = new {
```

Comment:
That's interesting, I had been adding full pocos for options but there's a lot more flexibility to using object and anonymous types, especially where we're not working too much with these objects in C#

Reply:
/shrug, honestly I'd much prefer using C# types for the options. Mainly as a guide because DataTables.js doesn't have great documentation.

I explored it, but unfortunately ended back up at object because DataTables.js has 3 different ways of utilizing this JS variable. (string array, object w/ properties, bool) blegh.

Don't get me wrong, DataTables.js is a great lib, but I'm glad for this Blazor implementation... so thank you @JustinWilkinson

Reply:
Yeah - I do remember problems like that. I added the `DiscriminatedUnion<T1, T2>` to handle some of this but I'm sure there's more to be done.